app/views/foreman_ansible/job_templates/ansible_windows_updates.erb

```diff
@@ -0,0 +1,187 @@
+<%#
+name: Ansible Windows - Updates
+snippet: false
+template_inputs:
+- name: blacklist
```

Comment:
```suggestion
- name: blocklist
```

https://community.theforeman.org/t/rfc-replacing-blacklist-with-blocklist-in-our-templates/19276

Reply:
In the ansible module it´s also called blacklist, so I think it would be good to leave it the same as in the ansible module

Reply:
ist im commit enthalten

Reply:
Done. Thanks.